### PR TITLE
request feedstock for QScreenCast

### PR DESCRIPTION
**Pull request by nerohmot**
Checklist
- [X] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [X] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example).
- [X] Source is from official source.
- [X] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [X] If static libraries are linked in, the license of the static library is packaged.
- [X] Build number is 0.
- [X] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [X] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [ ] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by andfoy** [comment]
@conda-forge/staged-recipes, this one is ready for review

File: recipes/QScreenCast/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/QScreenCast-{{ version }}.tar.gz
+  sha256: 0bbaffb73905f3fb2a9f73f00ff2e6f32bc3ee5c663ef9a3d346ef5ee0153807
+
+build:
+  number: 0
+  script: {{ PYTHON }} -m pip install . -vv
+  skip: True  # [not win]
```
Comment (ocefpaf) [inline recipes/QScreenCast/meta.yaml:16]: Looking at the code I did not find anything that is Windows only but they do declare it as the only platform :-/

Comment (andfoy) [reply recipes/QScreenCast/meta.yaml:16]: Thanks for the quick review @ocefpaf! I think the package has some problems with the Qt version available in conda-forge, which does not have support for QMultimedia calls in Mac nor Linux.

Comment (ocefpaf) [reply recipes/QScreenCast/meta.yaml:16]: Makes sense. Maybe we should open an issue in the feedstock to track that.

Comment (nerohmot) [reply recipes/QScreenCast/meta.yaml:16]: @ocefpaf, I made an issue for this :-)

**Pull request merged**